steamworks/interfaces/userstats.py

```
@@ -0,0 +1,161 @@
+        :param callback: callable
+        :param override_callback: bool
+        :return: bool
+        """
+        if callback:
+            if self._LeaderboardFindResult and override_callback:
+                self.SetFindLeaderboardResultCallback(callback)
+
+        else:
+            self.SetFindLeaderboardResultCallback(callback)
+
+        Steam.cdll.Leaderboard_FindLeaderboard(name.encode())
```

Ruff (0.14.5)

`Steam` may be undefined, or defined from star imports

(F405)